### ISSUE TITLE
Accounts, Blocks and Transactions tabs should remain visible

### ISSUE THREAD

**Issue by ivpavici** [0xSpaceShard/starknet-rivet#81]
...once you enter a individual block or account:

![image](https://github.com/user-attachments/assets/c6175fbf-3673-48e4-989c-9c5556fbc1e0)

**Comment by mirko-sisko** [comment]
I don't think this is necessary since it's only one level down, we have the Back button for returning to the home tab. What do you think @dsperac?
